packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -255,7 +254,29 @@ class CreateCommand extends CreateBase {
     }
 
     final String dartSdk = globals.cache.dartSdkBuild;
-    final bool includeIos = featureFlags.isIOSEnabled && platforms.contains('ios');
+    final bool includeIos;
+    final bool includeAndroid;
+    final bool includeWeb;
+    final bool includeLinux;
+    final bool includeMacos;
+    final bool includeWindows;
+    if (template == FlutterProjectType.module) {
+      // The module template only supports iOS and Android.
+      includeIos = true;
```

Comment:
should we still check if platforms.contains('ios')?

Reply:
We could - but it would change the current behaviour: https://github.com/flutter/flutter/pull/106300/files#diff-fa49e752dc7aba132fcab72c29e71c50541d612363171837a4658e0317fb7c9aL457